compiler/test/dotty/tools/StdLibSources.scala

```diff
@@ -0,0 +1,33 @@
+package dotty.tools
+
+import java.io.File
+
+import scala.io.Source
+
+object StdLibSources {
+
+  private final val stdLibPath = "../scala-scala/src/library/"
+
+  def blacklistFile: String = "./test/dotc/scala-collections.blacklist"
+
+  def whitelisted: List[String] = (all.toSet -- blacklisted).toList
```

Comment:
Added `useExplicitWhiteList` in `StdLibSources.scala` to allow the use of a whitelist. In the current scheme if `useExplicitWhiteList == true` and the whitelist file does not exist, it will be generated from the blacklist. If it exists it will simply use it.